### PR TITLE
exposing the biases for multi-class logistic regression

### LINKED ISSUE #1061: Our linear models are unbiased (at least in public)
You can extract weights out of the `MulticlassLogisticRegressionPredictor` now:

```c#
MulticlassLogisticRegressionPredictor predictor;
VBuffer<float>[] weights = null;
predictor.GetWeights(ref weights, out int numClasses);
```

There is no way to extract `_biases` out of the predictor though.  And we should have a way.
There's an `internal` method that accessed biases, but no `public` method to do so.

### PR DESCRIPTION
**Pull request by sfilipi**
Resolves #1061

### CODE REVIEW

**Comment by sfilipi** [comment]
@Zruty0  is this all the bug was about?

File: test/Microsoft.ML.StaticPipelineTesting/Training.cs [hunk 1]
```diff
@@ -285,6 +285,9 @@ public void SdcaMulticlass()
             Assert.NotNull(pred);
             VBuffer<float>[] weights = default;
             pred.GetWeights(ref weights, out int n);
+
+            var biases = pred.GetBiases();
```
Comment (sfilipi) [inline test/Microsoft.ML.StaticPipelineTesting/Training.cs]: Add Assert

**Comment by Zruty0** [comment]
Yes

---
In reply to: [428839240](https://github.com/dotnet/machinelearning/pull/1224#issuecomment-428839240) [](ancestors = 428839240)

File: test/Microsoft.ML.StaticPipelineTesting/Training.cs (hunk 1 above)
Comment (Ivanidzo4ka) [inline test/Microsoft.ML.StaticPipelineTesting/Training.cs]: >GetBiases [](start = 30, length = 9)

Is where are case in which user want's only weights or only biases? If no, I would suggest to rename GetWeights to GetWeightsAndBiases

Comment (sfilipi) [reply test/Microsoft.ML.StaticPipelineTesting/Training.cs]: let's leave them decoupled; it's usually more flexible. 

---
In reply to: [224521992](https://github.com/dotnet/machinelearning/pull/1224#discussion_r224521992) [](ancestors = 224521992)

**Comment by Ivanidzo4ka** [comment]
>             pred.GetWeights(ref weights, out int n);

You don't like idea to update CookbookSamples.cs as well? :(

---
In reply to: [429045145](https://github.com/dotnet/machinelearning/pull/1224#issuecomment-429045145) [](ancestors = 429045145)

---
Refers to: test/Microsoft.ML.StaticPipelineTesting/Training.cs:287 in 78841b9. [](commit_id = 78841b957c06e96e8996e57f0a8e363e66a14c8d, deletion_comment = False)

**Comment by sfilipi** [comment]
>             pred.GetWeights(ref weights, out int n);

Updated. 

---
In reply to: [429055124](https://github.com/dotnet/machinelearning/pull/1224#issuecomment-429055124) [](ancestors = 429055124,429045145)

---
Refers to: test/Microsoft.ML.StaticPipelineTesting/Training.cs:287 in 78841b9. [](commit_id = 78841b957c06e96e8996e57f0a8e363e66a14c8d, deletion_comment = False)

**Pull request merged**